src/components/molecules/Slider/style.scss

```diff
@@ -0,0 +1,69 @@
+.slider {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+
+    &__slide {
+        position: relative;
+        min-width: 100%;
+        height: auto;
+        overflow: hidden;
+
+        &__img {
+            width: 100%;
+        }
+    }
+
+    &__slide0-transitionOn {
```

Comment:
please get rid of 0, 100, 200 in classname. Thae best approach in your case if you have all sides in dom, you can use computed styles inside react component. In this case inline styles is not prohibited

Reply:
Ok, changed